go.mod

```diff
@@ -281,6 +281,7 @@ require (
 )
 
 replace (
+	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
```

Comment:
does this need to be added into the upgrading.md?

Reply:
I don't think so, given all previous versions have this replace too:
https://github.com/cosmos/cosmos-sdk/blob/release/v0.46.x/go.mod#L162-L163
https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/go.mod#L142-L143
Maybe, we should have a section in the docs (but no UPGRADING.md) (https://docs.cosmos.network/main/migrations/intro) that chains are expected to use the same replace directive that in `simapp`.